addons/storysource/src/StoryPanel.js

```diff
@@ -4,7 +4,7 @@ import { styled } from '@storybook/theming';
 import { Link } from '@storybook/router';
 import { SyntaxHighlighter } from '@storybook/components';
 
-import createElement from 'react-syntax-highlighter/create-element';
+import createElement from 'react-syntax-highlighter/dist/esm/create-element';
```

Comment:
should be checked again based on the comment in https://github.com/conorhastings/react-syntax-highlighter/pull/141#issuecomment-537125730